tests/e2e/test_nvfp4_training.py

```
@@ -0,0 +1,1372 @@
+        class FakeVL(nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.model = nn.Module()
+                self.model.visual = VisionModel()
+                self.lm_head = nn.Linear(2048, 1024)
+
+        m = FakeVL().cuda().bfloat16()
+        for p in m.parameters():
+            p.requires_grad_(False)
+
+        name, vt = _find_vision_tower(m)
```

Ruff (0.15.15)

[warning] Unpacked variable `vt` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)